commcare_connect/organization/decorators.py

```diff
@@ -57,3 +60,37 @@ def _inner(request, *args, **kwargs):
         return view_func(request, *args, **kwargs)
 
     return _inner
+
+
+def opportunity_for_org_required(view_func):
```

Comment:
This could simply be called `opportunity_required`, all opps are associated with an org.